packages/dialog/dialog-wrapper.md

```diff
@@ -0,0 +1,86 @@
+## Description
+
+`sp-dialog` display important information that users need to acknowledge. They appear over the interface and block further interactions.
+
+### Installation
+
+[![See it on NPM!](https://img.shields.io/npm/v/@spectrum-web-components/dialog?style=for-the-badge)](https://www.npmjs.com/package/@spectrum-web-components/dialog)
+[![How big is this package in your project?](https://img.shields.io/bundlephobia/minzip/@spectrum-web-components/dialog?style=for-the-badge)](https://bundlephobia.com/result?p=@spectrum-web-components/dialog)
+
+```
+npm install @spectrum-web-components/dialog
```

Comment:
^ Same as above

Reply:
Here there is only the `@spectrum-web-components/dialog`, you can choose to use one component or the other, but both are included.